### ISSUE TITLE
Package request: Update existing libspnav to 1.1

### ISSUE THREAD

**Issue by CtrlC-Root** [void-linux/void-packages#44550]
### Package name

libspnav

### Package homepage

https://github.com/FreeSpacenav/libspnav

### Description

I want to make PRs to package [spacenavd](https://github.com/FreeSpacenav/spacenavd) and [spnavcfg](https://github.com/FreeSpacenav/spnavcfg). These depend on [libspnav](https://github.com/FreeSpacenav/libspnav) which is [currently packaged](https://github.com/void-linux/void-packages/blob/master/srcpkgs/libspnav/template) but out of date. Some updates on the package:

* The releases are now available on GitHub (see links above).
* The new version supports a non-X11 build as well as the X11 build. It would be nice to have both as variants although I'm not familiar enough with XBPS to know if this is possible or how it would work.

Heads up @yopito since you are the current maintainer and was the last one to update the libspnav package.

Also linking a previous PR to package spacenavd at https://github.com/void-linux/void-packages/pull/19269 which seems to have been closed for some reason.

### Does the requested package meet the package requirements?

Compiled, Required

### Is the requested package released?

Yes

**Comment by CtrlC-Root** [comment]
I see that but it's not clear to me what to do when the change is more than updating to a new version. Did I miss some part of the documentation that talks about the process for this? Specifically:

>     * The releases are now available on GitHub (see links above).

And Sourceforge no longer has the newer release. So it's not just a package version update.

![image](https://github.com/void-linux/void-packages/assets/4819590/eff2963b-a9c4-4151-ba3f-e6ae1c7205bf)

>     * The new version supports a non-X11 build as well as the X11 build. It would be nice to have both as variants although I'm not familiar enough with XBPS to know if this is possible or how it would work.

Should these be separate packages? How does this work in XBPS?

**Comment by CtrlC-Root** [comment]
To clarify, given FreeCAD builds on top of libspnav, updating the library and packaging the necessary tools to actually use the functionality will likely require 3+ PRs. That could take weeks (https://github.com/void-linux/void-packages/pull/44235). That is a lot of work to do with no discussion, guidance, or expectation of it being accepted beforehand. What is the appropriate process to engage with the community around this kind of work before starting on it?
